store/copr/coprocessor.go

```diff
@@ -72,6 +73,8 @@ const (
 	smallTaskSigma         = 0.5
 )
 
+var smallTaskConcurrencyLimit = 20 * runtime.NumCPU()
```

Comment:
It is dangerous. If tidb is in container. The runtime.NumCPU() will return the cpu number of the physical machine. In fact, you need to the value in the cgroup.

Reply:
Please use ```runtime.GOMAXPROCS(0)``` after ```init``` have been called.